psiturk/psiturk_shell.py

```diff
@@ -941,60 +944,89 @@ def hit_create(self, numWorkers, reward, duration):
             print '*** duration must be greater than 0'
             return
 
-        # register with the ad server (psiturk.org/ad/register) using POST
-        if os.path.exists('templates/ad.html'):
-            ad_html = open('templates/ad.html').read()
-        else:
-            print '\n'.join(['*****************************',
-                             '  Sorry, there was an error registering ad.',
-                             '  Both ad.html is required to be in the templates folder',
-                             '  of your project so that these Ad can be served!'])
-            return
+        if use_psiturk_ad_server:
 
-        size_of_ad = sys.getsizeof(ad_html)
-        if size_of_ad >= 1048576:
-            print '\n'.join(['*****************************',
-                             '  Sorry, there was an error registering ad.',
-                             '  Your local ad.html is %s byes, but the maximum',
-                             '  template size uploadable to the Ad server is',
-                             '  1048576 bytes!' % size_of_ad])
-            return
+            # register with the ad server (psiturk.org/ad/register) using POST
+            if os.path.exists('templates/ad.html'):
+                ad_html = open('templates/ad.html').read()
+            else:
+                print '\n'.join(['*****************************',
+                                 '  Sorry, there was an error registering ad.',
+                                 '  Both ad.html is required to be in the templates folder',
```

Comment:
Might be worthwhile to update this comment to be a little clearer -- "both" doesn't make much sense in this context. So maybe something like "Sorry, there was an error registering the ad. The file ad.html is required to be in the templates folder of your project so that the ad can be served!"